migrations/20230210165254_assessment_instances__include_in_statistics__update.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE assessment_instances AS ai
+SET include_in_statistics = NOT users_is_instructor_in_course_instance(ai.user_id, a.course_instance_id)
+FROM assessments AS a
+WHERE
+    ai.assessment_id = a.id
+    AND ai.date > now() - interval '4 months';
```

Comment:
Checking my understanding: this line plus the fact that `include_in_statistics` defaults to `TRUE` means that we'll have slightly inaccurate stats for old course instances? I think that's a reasonable tradeoff.

Reply:
Yep, that's right. I intend to run the migration manually on older course instances, but I didn't want to put that into the migration. I might shorten the migration to 2 months, actually.

Reply:
Actually I timed it and it looks like doing all `assessment_instances` at once will only take about 15 seconds, so in 1b5299e2e3578171ad8218d7d5e4fbc3d8591f09 I removed the date WHERE clause.